packages/mdc-ripple/util.js

```diff
@@ -97,31 +97,46 @@ function supportsCssVariables(windowObj, forceRefresh = false) {
  * Determine whether the current browser supports passive event listeners, and if so, use them.
  * @param {!Window=} globalObj
  * @param {boolean=} forceRefresh
- * @return {boolean|{passive: boolean}}
+ * @return {boolean|!EventListenerOptions}
  */
 function applyPassive(globalObj = window, forceRefresh = false) {
   if (supportsPassive_ === undefined || forceRefresh) {
     let isSupported = false;
     try {
       globalObj.document.addEventListener('test', null, {get passive() {
         isSupported = true;
+        return isSupported;
       }});
     } catch (e) { }
 
     supportsPassive_ = isSupported;
   }
 
-  return supportsPassive_ ? {passive: true} : false;
+  return supportsPassive_
+    ? /** @type {!EventListenerOptions} */ ({passive: true})
+    : false;
 }
 
 /**
  * @param {!Object} HTMLElementPrototype
- * @return {!Array<string>}
+ * @return {string}
  */
 function getMatchesProperty(HTMLElementPrototype) {
-  return [
-    'webkitMatchesSelector', 'msMatchesSelector', 'matches',
-  ].filter((p) => p in HTMLElementPrototype).pop();
+  /**
```

Comment:
I'm a little unclear on how changing this function body is related to the purpose of this PR... AFAICT it doesn't change the behavior of what's returned.

Reply:
The original function had to guarantee of returning a string since `filter` can return an empty array which causes `pop` to return undefined. We know it will always return a string since every HTMLElement has one of those properties but a type system doesn’t know that. I’m happy to refactor this if you have suggestions!